apps/dcos_overlay/src/dcos_overlay_poller.erl

```diff
@@ -339,6 +501,36 @@ parse_vtep_mac(MAC) ->
         end,
         MACComponents).
 
+%%%===================================================================
+%%% Metrics functions
+%%%===================================================================
+
+-spec(init_metrics() -> ok).
+init_metrics() ->
+    prometheus_counter:new([
+        {registry, overlay},
+        {name, polls_total},
+        {help, "Total number of times local Mesos overlays have beed polled."}]),
+    prometheus_histogram:new([
```

Comment:
ditto: summary

Actually, the poller calls `poll/1` function every 30-120 seconds, telegraf gets metrics from dcos-net every 30 seconds. so `sum` time series will have all information about each individual polling call.